### PR TITLE
Map menu dropdown

### PR DESCRIPTION
**Pull request by alecavi**
This MR adds dropdown menus to each and every map, automatically generated from the map data.

### CODE REVIEW

File: Software_Code/Goals/Clean-Energy.php [hunk 1]
```diff
@@ -147,58 +147,17 @@
...
-              popUp.style.display = "none";
-            }
-          }
-
-        </script>
-
-        <button type="button" class="btn btn-dark" onclick="popUpWindow();">More info</button>
-        <div class="infoPopup" id="infoPage" style="display: none;" >
-          <form action="/No-Poverty.php" class="infoContainer">
-          <?php
```
Comment (DanielJakubek) [inline Software_Code/Goals/Clean-Energy.php:194]: This needs to stay, put table this in please

**Review by DanielJakubek — CHANGES_REQUESTED** [review]
Yup, looks okay apart from the lack of comments in the js and php. Will also need to add back the popups back to each page.

**Comment by ScottyDog5** [comment]
Other than needing some comments it looks good

**Review by DanielJakubek — APPROVED** [review]
Looks good now but you still need to add comments.

**Pull request merged**